featuretools/utils/wrangle.py

```diff
@@ -168,3 +169,11 @@ def _is_url(string):
     Returns a boolean.
     '''
     return 'http' in string
```

Comment:
Not a big deal but, should we change the `_is_s3` to be (same for `_is_url`)
```python
def _is_s3(string):	
    '''	
    Checks if the given string is an url path.	
    Returns a boolean.	
    '''	
return string.startswith("s3://")
```

Reply:
for is url are you thinking `string.startswith("http")` ?

Reply:
Yes.

Reply:
Sounds good to me